src/components/10-atoms/checkbox/index.js

```diff
@@ -170,6 +200,32 @@ class AXACheckbox extends NoShadowDOM {
     }
   }
 
+  // react has no innerHTML in constructor
+  shouldUpdate() {
+    const {
+      isReact,
+      hasChildren,
+      label,
+      labelTextElement,
+      innerHTML,
+      iteration,
+    } = this;
+
+    // if react & innerHTML is set or if is react and has a label and innerHTML
+    if (
+      isReact &&
+      innerHTML !== '' &&
+      !hasChildren &&
+      !labelTextElement &&
+      (!label || (label && iteration === 1))
+    ) {
+      this.wrapChildren();
+    }
+
+    this.iteration += 1;
+    return true;
```

Comment:
Why do you return true?

Reply:
It needs it to proceed the update
https://lit-element.polymer-project.org/guide/lifecycle#shouldupdate